airflow/providers/apache/kylin/operators/kylin.py

```diff
@@ -0,0 +1,171 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import time
+from datetime import datetime
+from typing import Optional
+
+from kylinpy import kylinpy
+
+from airflow.exceptions import AirflowException
+from airflow.models import BaseOperator
+from airflow.providers.apache.kylin.hooks.kylin import KylinHook
+from airflow.utils import timezone
+from airflow.utils.decorators import apply_defaults
+
+
+class KylinOperator(BaseOperator):
```

Comment:
```suggestion
class KylinSubmitRequestOperator(BaseOperator):
```
https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#naming-conventions-for-provider-packages

Reply:
@shaofengshi @zhaoyongjie @nichunen @wangrupeng @zhangayqian @hit-lacus Do you have any suggestions？Thanks！

Reply:
@mik-laj hi Mik, thanks for reviewing the PR! This is Shaofeng Shi, I'm one of Apache Kylin PMC & committer; Yongheng is my team member, this Airflow Operator has been developed and deployed for a while. Regarding the naming convention, I suggest to name it as "KylinCubeOperator", because the actions in it are related with Cubes (not just submit request). Hope it can be accepted. Thanks!

Reply:
+1 , `SubmitRequest` is too general/wide from my side, use `KylinCubeOperator` maybe better because almost all requests are interacting with **Cube**.

Reply:
I hive renamed it to ‘KylinCubeOperator’